analyzers/tests/SonarAnalyzer.Test/Helpers/KnownAssemblyTest.cs

```diff
@@ -255,103 +255,115 @@ public void Combinator_StartsWith_Start_Or_EndsWith_End_2(string name, bool expe
 
     [TestMethod]
-    public void FluentAssertions_6_10()
-    {
-        var compilation = TestHelper.CompileCS("// Empty file", NuGetMetadataReference.FluentAssertions("6.10.0").ToArray()).Model.Compilation;
-        compilation.References(KnownAssembly.FluentAssertions).Should().BeTrue();
-    }
+    public void FluentAssertions_6_10() =>
+        CompilationShouldReference(NuGetMetadataReference.FluentAssertions("6.10.0").ToArray(), KnownAssembly.FluentAssertions);
 
     [TestMethod]
-    public void FluentAssertions_NoReference()
-    {
-        var compilation = TestHelper.CompileCS("// Empty file").Model.Compilation;
-        compilation.References(KnownAssembly.FluentAssertions).Should().BeFalse();
-    }
+    public void FluentAssertions_NoReference() =>
+        CompilationShouldNotReference(KnownAssembly.FluentAssertions);
 
     [TestMethod]
-    public void NFluent_2_8()
-    {
-        var compilation = TestHelper.CompileCS("// Empty file", NuGetMetadataReference.NFluent("2.8.0").ToArray()).Model.Compilation;
-        compilation.References(NFluent).Should().BeTrue();
-    }
+    public void NFluent_2_8() =>
+        CompilationShouldReference(NuGetMetadataReference.NFluent("2.8.0").ToArray(), NFluent);
 
     [TestMethod]
-    public void NFluent_1_0()
-    {
+    public void NFluent_1_0() =>
         // 1.0.0 has no publicKeyToken
-        var compilation = TestHelper.CompileCS("// Empty file", NuGetMetadataReference.NFluent("1.0.0").ToArray()).Model.Compilation;
-        compilation.References(NFluent).Should().BeTrue();
-    }
+        CompilationShouldReference(NuGetMetadataReference.NFluent("1.0.0"), NFluent);
 
     [TestMethod]
-    public void NFluent_NoReference()
-    {
-        var compilation = TestHelper.CompileCS("// Empty file").Model.Compilation;
-        compilation.References(NFluent).Should().BeFalse();
-    }
+    public void NFluent_NoReference() =>
+        CompilationShouldNotReference(NFluent);
 
     [TestMethod]
-    public void NSubstitute_5_0()
-    {
-        var compilation = TestHelper.CompileCS("// Empty file", NuGetMetadataReference.NSubstitute("5.0.0").ToArray()).Model.Compilation;
-        compilation.References(NSubstitute).Should().BeTrue();
-    }
+    public void NSubstitute_5_0() =>
+        CompilationShouldReference(NuGetMetadataReference.NSubstitute("5.0.0"), NSubstitute);
 
     [TestMethod]
-    public void NSubstitute_NoReference()
-    {
-        var compilation = TestHelper.CompileCS("// Empty file").Model.Compilation;
-        compilation.References(NSubstitute).Should().BeFalse();
-    }
+    public void NSubstitute_NoReference() =>
+        CompilationShouldNotReference(NSubstitute);
+
+    [TestMethod]
+    public void MicrosoftExtensionsLoggingAbstractions_Latest() =>
+        CompilationShouldReference(NuGetMetadataReference.MicrosoftExtensionsLoggingAbstractions(Constants.NuGetLatestVersion), MicrosoftExtensionsLoggingAbstractions);
```

Comment:
`Constants.NuGetLatestVersion` is the default value and can be omitted.
```suggestion
        CompilationShouldReference(NuGetMetadataReference.MicrosoftExtensionsLoggingAbstractions(), MicrosoftExtensionsLoggingAbstractions);
```

Reply:
For the others as well.